README.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Social Rest API
+===============
+
+Social Rest API is a Django app to conduct construction of APIs ment to be used for on-demannd and social apps.
+
+Detailed documentation is in the "docs" directory.
+
+Quick start
+-----------
+
+1. Add "social-rest-api" to your INSTALLED_APPS setting like this::
+
+    INSTALLED_APPS = [
+        ...
+        'social-rest-api',
+    ]
+
+2. Include the Social Rest API URLconf in your project urls.py like this::
+
+    path('social-rest-api/', include('social-rest-api.urls')),
```

Comment:
we will probably have to change this url for something simpler, like `path('socia/', include('social-rest-api.urls'))`

Reply:
Makes sense. And we should also close the discussion about social vs on-demand